docs/deprecations.md

```diff
@@ -363,3 +363,11 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 Deprecation code: `BIN_SERVERLESS`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+<a name="AWS_API_GATEWAY_NON_APPLICABLE_SETTINGS"><div>&nbsp;</div></a>
```

Comment:
Sorry for not noticing it before - it should be listed as first on the list, right after `How to disable specific deprecation logs` section